tensorflow/core/common_runtime/next_pluggable_device/BUILD

```diff
@@ -99,6 +99,20 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "next_pluggable_device_factory_hdrs",
+    hdrs = ["next_pluggable_device_factory.h"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":next_pluggable_device_api",
+        "//tensorflow/c:tf_status_headers",
+        "//tensorflow/c:tf_status_helper",
```

Comment:
This causes some tests to fail (there were some recent changes about tf_status_helper). After removing `"//tensorflow/c:tf_status_helper",` it should be good to go!

Reply:
@jyingl3  done. Thanks